jsk_mbzirc_common/gazebo_model/world/mbzirc_arena.world

```diff
@@ -32,10 +32,16 @@
     </include>
 
     <include>
-      <pose>0 0 0 1.57 -0 0</pose>
+      <pose>0 0 0 1.5708 -0 1.5708</pose>
```

Comment:
what is the "correct map direction"?
please try these files in this PR (
https://github.com/tongtybj/mbzirc/pull/23) and if you thin you have to
change direction, please change 4 launch files and also you need to change
gazebo plugins.
mbzirc_arena.launch  mbzirc_task_1.launch  mbzirc_task_2.launch
 mbzirc_task_3.launch


--
◉ Kei Okada


On Wed, Jan 13, 2016 at 11:36 PM, Chen <notifications@github.com> wrote:

> In jsk_mbzirc_common/gazebo_model/world/mbzirc_arena.world
> <https://github.com/tongtybj/mbzirc/pull/21#discussion_r49596788>:
>
> > @@ -32,10 +32,16 @@
> >      </include>
> >
> >      <include>
> > -      <pose>0 0 0 1.57 -0 0</pose>
> > +      <pose>0 0 0 1.5708 -0 1.5708</pose>
>
> Because in my gazebo, I have to change the yaw to get the correct map
> direction... 2016-01-13 22:33 GMT+08:00 Kei Okada <
> notifications@github.com>:
> … <#-1870602016_>
> In jsk_mbzirc_common/gazebo_model/world/mbzirc_arena.world <#21 (comment)
> <https://github.com/tongtybj/mbzirc/pull/21#discussion_r49596385>>: > @@
> -32,10 +32,16 @@ > </include> > > <include> > - <pose>0 0 0 1.57 -0
> 0</pose> > + <pose>0 0 0 1.5708 -0 1.5708</pose> why you changed direction,
> i reverted this at 0bddf7f
> <https://github.com/tongtybj/mbzirc/commit/0bddf7f42e9603097c19e5646757b904eb910dec>
> <0bddf7f
> <https://github.com/tongtybj/mbzirc/commit/0bddf7f42e9603097c19e5646757b904eb910dec>>
> — Reply to this email directly or view it on GitHub <
> https://github.com/tongtybj/mbzirc/pull/21/files#r49596385>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tongtybj/mbzirc/pull/21/files#r49596788>.
>